docs/API.md

```diff
@@ -1591,7 +1661,7 @@ whose series is 2. Available filters are:
 * promulgated - the charm has been promulgated.
 * provides - interfaces provided by the charm.
 * requires - interfaces required by the charm.
-* series - the charm's series.
+* series - the set of series associated with the charm.
```

Comment:
a series supported by the charm

?

(this endpoint is used, and I don't think its semantics need to change)